presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -360,6 +366,11 @@ protected RowExpression visitFunctionCall(FunctionCall node, Void context)
         @Override
         protected RowExpression visitSymbolReference(SymbolReference node, Void context)
         {
+            Integer field = layout.get(Symbol.from(node));
```

Comment:
Can we name this something else? It's confusing to see `field(field...)`

Reply:
`channel` maybe